docs/machine-learning/tutorials/sentiment-analysis.md

```diff
@@ -56,6 +61,22 @@ Predict the **sentiment** of a new website comment, either positive or negative,
 * Please refrain from adding nonsense to Wikipedia.
 * He is the best, and the article should say that.
 
+Classification type of task is best suited for this
+
+### About the classification model
+
+Classification is a machine learning method that uses data to **determine** the category, type, or class of an item or row of data. For example, you can use classification to:
+
+* Identify sentiment as positive or negative.
+* Classify email filters as spam, junk, or good.
```

Comment:
"classify email filters" Should it be "classify email"? (you're classifying the email, not the filters.)